graphs/kickiip.fbp

```diff
@@ -1,4 +1,5 @@
-# !runtime noflo
+# @runtime noflo-nodejs
+# @name IIPKickTest
```

Comment:
If the library had been using grunt-noflo-manifest, these would've been needed already long time ago.